app/models/manageiq/providers/container_manager.rb

```diff
@@ -67,18 +68,6 @@ def supports_metrics?
       true
     end
 
-    # required by aggregate_hardware
-    alias :all_computer_systems :computer_systems
-    alias :all_computer_system_ids :computer_system_ids
-
-    def aggregate_cpu_total_cores(targets = nil)
```

Comment:
can you add back using computer_system_hardwares?

Reply:
think on chat @d-m-u said this doesn't work

Reply:
i think this has_many works fine on container manager. i do not think it works for container project.